docs/UserGuide.md

```diff
@@ -58,10 +58,10 @@ Assistant who can type fast, TAilor can get your contact management tasks done f
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
```

Comment:
For this part on the features, could you refer to the original code / [User Guide](https://ay2122s2-cs2103t-w12-1.github.io/tp/UserGuide.html) to maintain the separation of commands? (common commands / contact list commands / task list commands)

Reply:
> 

one of his methods is model.getMod(...) but it resembles a getter function for a class that doesn't have that attribute. So it is ultimately slightly misleading. The method's previous name was `retreiveMod(...)`. May have to look into this for people who try to understand our implementations from the DG's UML diagrams first rather than the codebase